pkg/satellite/satellitedb/projects.go

```diff
@@ -43,10 +43,24 @@ func (projects *projects) GetByOwnerID(ctx context.Context, ownerID uuid.UUID) (
 func (projects *projects) GetByUserID(ctx context.Context, userID uuid.UUID) ([]satellite.Project, error) {
 	projectsDbx, err := projects.db.All_Project_By_ProjectMember_MemberId(ctx, dbx.ProjectMember_MemberId(userID[:]))
 	if err != nil {
-		return nil, err
+		if checkNoRowsErr(err) {
```

Comment:
You can use `err != nil && checkNoRowsErr(err)`  or alternatively check for nil inside `checkNoRowsErr`.